src/main/resources/assets/styles/main.scss

```diff
@@ -136,3 +136,8 @@ section.part-divider + section.part-divider {
     width: 95%;
   }
 }
+
+// Highcharts Keyboard navigation tab fix (accessibility elements fix)
+.highcharts-a11y-proxy-button.highcharts-no-tooltip{
```

Comment:
Try to move this to _highchart.scss instead :)

Reply:
ok